### PR TITLE
Ansible 4 roadmap

### PR DESCRIPTION
**Pull request by abadger**
##### SUMMARY
Schedule for the release of Ansible-4.x

This is going to be discussed and approved at an Ansible Community IRC Meeting.

##### ISSUE TYPE
- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by dmsimard — APPROVED** [review]
Thanks for putting this together!

Looks good to me -- it even includes a link to the 2.11 roadmap.

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+.. contents::
+   :local:
+
+
+Release schedule
+=================
+
+
+:2021-01-26: New Collections will be reviewed for inclusion in ansible-4.
+:2021-03-03: ansible-4.0.0 alpha1 (biweekly alpha releases.  This is timed to coincide with the start of the ansible-core-2.11 pre-releases)
```
Comment (gundalow) [inline docs/docsite/rst/roadmap/COLLECTIONS_4.rst]:
```suggestion
:2021-03-03: ansible-4.0.0 alpha1 (The ``ansible`` alphas are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases)
```

Comment (abadger) [reply docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: I'd like to keep biweekly as all the other pre-releases are released weekly.  Maybe:

```suggestion
:2021-03-03: ansible-4.0.0 alpha1 (biweekly ``ansible`` alphas.  These are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases)
```

Comment (gundalow) [reply docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: Sounds good

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   :local:
+
+
+Release schedule
+=================
+
+
+:2021-01-26: New Collections will be reviewed for inclusion in ansible-4.
+:2021-03-03: ansible-4.0.0 alpha1 (biweekly ``ansible`` alphas.  These are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases)
+:2021-03-17: ansible-4.0.0 alpha2
```
Comment (dericcrago) [inline docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: were these two dates intentionally on Wednesday or should they be shifted to Tuesday?

```suggestion
:2021-03-02: ansible-4.0.0 alpha1 (biweekly ``ansible`` alphas.  These are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases)
:2021-03-16: ansible-4.0.0 alpha2
```

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+:2021-03-30: ansible-4.0.0 alpha3
+:2021-04-13: last day for new collections to be submitted for inclusion in ansible-4. Note that collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
+:2021-04-13: ansible-4.0.0 alpha4
+:2021-04-14: Community IRC Meeting topic: list any new collection reviews which block release.  list any backwards incompatible collection releases that beta1 should try to accommodate.
+:2021-04-21: Community IRC Meeting topic: Decide what contingencies to activate for any blockers that do not meet the deadline.
+:2021-04-26: last day for new collections to be **reviewed and approved** for inclusion in ansible-4
+:2021-04-26: last day for collections to make backwards incompatible releases that will be accepted into Ansible-4.
+:2021-04-27: ansible-4.0.0 beta1 -- feature freeze [1]_ (weekly beta releases.  Collection owners and interested users
+             should test for bugs)
```
Comment (dericcrago) [inline docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: extra whitespace?

```suggestion
:2021-04-27: ansible-4.0.0 beta1 -- feature freeze [1]_ (weekly beta releases.  Collection owners and interested users should test for bugs)
```

**Review by dericcrago — APPROVED** [review]

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst (hunk 3 above)
Comment (abadger) [reply docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: I cut lines at 100 characters for easy reading as text.  But rst will parse either one correctly when transforming to html.

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst (hunk 2 above)
Comment (abadger) [reply docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: Yes... It's easier to see when I have the ansible-core pre-releases interspersed: https://hackmd.io/y7BBcweNR3aRVLuMbKkDxw#Proposed-Schedule-for-Ansible-40   The idea is for the ansible alphas to release the day after the ansible-core betas and release candidcates.

Comment (abadger) [reply docs/docsite/rst/roadmap/COLLECTIONS_4.rst]: Okay, relrod has finalized the dates for alpha2 and so forth: https://github.com/ansible/ansible/pull/73560/files

ansible-core-2.11-beta1 will still be on a Tuesday but subsequent releases will be shifted to Monday.  So I'll change our ansible-4.0.0 alpha2 to be a Tuesday.

About why beta1 is on a Tuesday, it is a combination of publishing that date quite a few months ago so it seems too late to change and it gives people one extra day to get their work finished before the beta freeze.

**Review by acozine — APPROVED** [review]
Looks good.

File: docs/docsite/rst/roadmap/COLLECTIONS_4.rst [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+=================
+
+
+:2021-01-26: New Collections will be reviewed for inclusion in Ansible 4.
+:2021-03-03: Ansible-4.0.0 alpha1 (biweekly ``ansible`` alphas.  These are timed to coincide with the start of the ``ansible-core-2.11`` pre-releases).
+:2021-03-16: Ansible-4.0.0 alpha2
+:2021-03-30: Ansible-4.0.0 alpha3
+:2021-04-13: Last day for new collections to be submitted for inclusion in Ansible 4. Note that collections MUST be reviewed and approved before being included. There is no guarantee that we will review every collection. The earlier your collection is submitted, the more likely it will be that your collection will be reviewed and the necessary feedback can be addressed in time for inclusion.
+:2021-04-13: Ansible-4.0.0 alpha4
+:2021-04-14: Community IRC Meeting topic: list any new collection reviews which block release.  list any backwards incompatible collection releases that beta1 should try to accommodate.
```
Comment (samccann) [inline docs/docsite/rst/roadmap/COLLECTIONS_4.rst]:
```suggestion
:2021-04-14: Community IRC Meeting topic: list any new collection reviews which block release.  List any backwards incompatible collection releases that beta1 should try to accommodate.
```

**Review by samccann — APPROVED** [review]

**Pull request merged**